File: notes/loop_notes.py
#MW 7th LOOP NOTES

foods =["Chocolate", "Pasta","Brownies","Cookies","Garlic Bread","Fajitas"]

#What is a loop? 

#Repeats the code inside them for a specified number of times

#What are the two types of loops?

# For loop
for food in foods:
    #print(f"{food} is my favorite food!")
    print("Hi")

for x in range(1,21):
    print(x)

#While loops

#i = 0
#while True:
    #print(i)
    #i+=1
#Infininite loop

i=1


while i <= 20:
    print(i)
    i+=1
x=1

while x < 21:
    if x % 2 ==0:
     print(f"{x} is an even number.")
    else:
     print (f"{x} is an odd number")
    x+=1



import random

d=1
end = random.randint(1,20)

#while d!= end:
   #print("Duck")
   #d+=1

#print("Goose")
while True:
   if d==end:
      print("Goose")
      break
   else:
      print("Duck")
      d+=1





#What is iteration

#How many times a loop has run

#What are lists?

#  
#How do you make lists in python?

#
  
#How do you make for loops in python?
#
  
#How do you make while loops in python?

File: c_programming/notes/loop_notes.c
//MW 7th loop notes

#include <stdio.h>
#include <string.h>

int main (void){

    int nums[] = {1,2,3,4,5,6,7};
    char candies[3][20] = {"Skittles", "Hersheys", "KitKat"};             
    for(int x = 0; x <7; x++){
        printf("%d\n", nums[x]);
    }

    for(int i = 0; i <3; i++){
    printf("%s, is my favorite candy!\n", candies [i]);
    }
    for(int num = 1; num <11; num++){
    printf("%d\n", num);
    }

int num = 0;
while(num < 11){
    printf("%d\n", num);
    num++;
}


    return 0;
}


/*
What is a loop? 
it loops what is inside of it until you tell it to stop


What are the two types of loops?
for loop and while loop


What is iteration
the amount of times it loops


What are arrays?
its just a list, the variables are set next to eachother

How do you make lists in C?
 int nums[] = {1,2,3,4,5,6,7};


How do you make for loops in C? 
for(int x = 0; x <7; x++){
        printf("%d\n", nums[x]);
    }


How do you make while loops in C?

while(num < 11){
    printf("%d\n", num);
    num++;
}


*/